swagger2.0/oic.r.rule.expression.swagger.json

```diff
@@ -0,0 +1,153 @@
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": { "$ref": "#/definitions/ruleexpression-update" },
+            "x-example":
+              {
+                "ruleenable": true,
+                "actionenable": false
+              }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description" : "",
+            "x-example": {
+              "rule": "(timeperiod:triggered = true)",
+              "ruleresult": false,
+              "ruleenable": true,
+              "actionenable": false
+            },
+            "schema": { "$ref": "#/definitions/ruleexpression" }
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "interface-rw": {
+      "in": "query",
+      "name": "if",
+      "type": "string",
+      "enum": ["oic.if.rw"]
+    },
+    "interface-all": {
+      "in": "query",
+      "name": "if",
+      "type": "string",
+      "enum": ["oic.if.rw","oic.if.baseline"]
+    }
+  },
+  "definitions": {
+    "ruleexpression" : {
+      "properties": {
+        "rt": {
+          "description": "The Resource Type.",
+          "items": {
+            "enum": ["oic.r.rule.expression"],
+            "maxLength": 64,
+            "type": "string"
+          },
+          "minItems": 1,
+          "uniqueItems": true,
+          "readOnly": true,
+          "type": "array"
+        },
+        "rule": {
+          "description": "The logical expression to be evaluated, see BNF",
+          "type": "string"
+        },
+        "ruleresult": {
+          "type": "boolean",
+          "readOnly": true
+        },
+        "ruleenable": {
+          "type": "boolean"
+        },
+        "actionenable": {
+          "type": "boolean"
+        },
+        "n": {
+            "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/n"
+        },
+        "id": {
+            "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/id"
+        },
+        "if": {
+          "description": "The OCF Interface set supported by this Resource.",
+          "items": {
+            "enum": [
+              "oic.if.rw",
+              "oic.if.baseline"
+            ],
+            "type": "string"
+          },
+          "minItems": 2,
+          "uniqueItems": true,
+          "readOnly": true,
+          "type": "array"
+        }
+      },
+      "type": "object",
+      "required": ["rule","ruleresult","actionenable","ruleenable"]
+    },
+    "ruleexpression-update" : {
+      "properties": {
+        "rule": {
+          "description": "The logical expression to be evaluated, see BNF",
+          "type": "string"
+        },
+        "ruleenable": {
+          "type": "boolean"
+        },
+        "actionenable": {
+          "type": "boolean"
+        }
```

Comment:
Add "ruleresult" if you agree it should be RW.

Reply:
I did.  Can you delete this comment.